WooCommerce/Classes/ViewModels/Order Details/Refunded Products/RefundedProductCellViewModel.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import Yosemite
+
+
+// MARK: - View Model for a refunded product cell
+//
+struct RefundedProductCellViewModel {
```

Comment:
similar to the previous comment, it looks like this struct only has to expose `imageURL`, `name`, `quantity`, `price`, and `sku`. could you make sure the other properties are private?